apps/sms/views/shared/style/panel.css

```diff
@@ -9,56 +9,17 @@
 
   background: white;
 
-  animation: .2s both linear;
+  animation: var(--transition-duration) both ease;
 }
-html[dir="ltr"] .panel { transform: translate(100%); }
-html[dir="rtl"] .panel { transform: translate(-100%); }
 
-/**
- * adding html[dir] to make the specificity of this rule high enough to
- * overwrite the default translation for element with class panel just above,
- * without needing to put a !important
- */
-html[dir] .panel-active {
```

Comment:
Oops, I let slipped this one, sorry for the delay. I don't think it's a problem if it's hidden in the same position in both direction. But in this case, you might want to put a `/* BiDi proof */ comment just above for the reader (when RTL-izing an application, I usually execute a big sed marking every place where it needs checking. These comments speeds up this process because I don't need to ask myself why it is not symmetric).